chia/policy/fee_estimator.py

```diff
@@ -0,0 +1,41 @@
+import abc
```

Comment:
Hey @altendky is this the best way to enforce an interface in Pyton?

Reply:
I like `Protocol`.  https://docs.python.org/3.10/library/typing.html#typing.Protocol  Though we get it via `from typing_extensions import Protocol` since it was introduced in Python 3.8.